.github/workflows/dismiss-stale-pr-reviews.yml

```diff
@@ -18,7 +18,27 @@ jobs:
   dismiss_stale_approvals:
     runs-on: ubuntu-latest
     steps:
+      - name: Check Core Team Membership
+        env:
+          ACTOR: ${{ github.actor }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          TEAM_ID: '11759072'
+        run: |
+          curl -X GET \
+            https://api.github.com/teams/${TEAM_ID}/memberships/${ACTOR} \
+              -H "Authorization: Bearer ${GITHUB_TOKEN}" \
+              -H "Accept: application/vnd.github+json" \
+              -o response.json
+
+          if [ $? -eq 0 ]; then
+            echo "::set-output name=is_core_team::true"
+          else
+            echo "::set-output name=is_core_team::false"
```

Comment:
It looks like `set-output` is deprecated: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

I think this should be `echo "is_core_team=false" >> $GITHUB_OUTPUT`